pyanaconda/modules/payloads/source/cdrom/initialization.py

```diff
@@ -40,15 +40,23 @@ def _do_mount(self):
         """Run CD-ROM installation source setup."""
         log.debug("Trying to detect CD-ROM automatically")
 
+        device_name = self._search_for_optical_media()
+
+        if not device_name:
+            raise SourceSetupError("Found no CD-ROM")
+
+        return device_name
+
+    def _search_for_optical_media(self):
         device_tree = STORAGE.get_proxy(DEVICE_TREE)
         device_name = ""
 
         for dev_name in device_tree.FindOpticalMedia():
             try:
                 device_data = DeviceData.from_structure(device_tree.GetDeviceData(dev_name))
                 mount(device_data.path, self._target_mount, "iso9660", "ro")
-            except PayloadSetupError as e:
-                log.debug("Failed to mount %s: %s", dev_name, str(e))
+            except PayloadSetupError as ex:
```

Comment:
Why is `ex` better then `e`?

Reply:
It's solving `invalid-name` issue in my editor. I can return it back but I don't see a reason to do that.

Reply:
This is no outright PEP8 error,so I think we should keep the current overall style.